hack/config

```diff
@@ -12,6 +12,7 @@ NODE_DRIVER_REG_VERSION="v2.2.0"
 LIVENESS_PROBE_VERSION="v2.3.0"
 CSI_SNAPSHOT_VERSION="v4.2.1"
 CSI_SIG_STORAGE_PROVISIONER_VERSION="v2.2.1"
+KUBE_RBAC_PROXY_VERSION="4.10"
```

Comment:
@orenc1, Do you want to add it to the auto-bump bot?

Reply:
Not sure it is required. We explicitly specified only the y-stream version, so that we'll get automatic z-stream updates to the image.
We'll update this image tag once in a release.

Reply:
I think we need to update to 4.11 now ?